src/commands/create_gke_cluster.yml

```diff
@@ -8,7 +8,7 @@ parameters:
     description: "The name of the cluster"
   addons:
     type: string
-    default: "HorizontalPodAutoscaling,HttpLoadBalancing,Istio,CloudRun"
```

Comment:
I would imagine that leaving the beta would probably just be a patch update, but I think the removal of this so we can move this away from the beta...

If we want to keep the ability to use the beta, I think we should actually create a `beta` boolean parameter that we pass through to everything. However, I still think that we should make it **not** `beta` by default, which would still require the removal of this add-on and necessitate (IMHO) a major change.

If we're going to do a major change, I'm wondering if we should just move to the new include syntax with the scripts and do these changes all at once.

@lokst and @KyleTryon - as authors of the orb, I would love your thoughts here

Reply:
@dsayling I suspect this command may only have been available in `beta` at the time of the orb creation. I agree that we should make it not beta by default